src/classes/column.js

```diff
@@ -81,7 +81,7 @@ class Column {
         }
         
         if(this.sortableAsFloat){
-            return parseFloat(objectPath.get(row, this.field));
+            return parseFloat(get(row, this.field));
```

Comment:
Big +1, it is great for [tree shaking](https://developer.mozilla.org/fr/docs/Glossaire/Tree_shaking)